PHPCompatibility/Sniffs/Constants/RemovedConstantsSniff.php

```diff
@@ -28,6 +28,9 @@ class RemovedConstantsSniff extends AbstractRemovedFeatureSniff
      * The array lists : version number with false (deprecated) or true (removed).
      * If's sufficient to list the first version where the constant was deprecated/removed.
      *
+     * Optional, the array can contain an `alternative` key listing an alternative constant
+     * to be used instead.
+     *
```

Comment:
Am I missing something here or is this not used anywhere ?

Reply:
This was a feature of the parent class which was so far unused by this sniff. If the `alternative` key exists (as it does for the constant added in this PR), the parent sniff will automatically add `Use ... instead` at the end of the error message.

That's also why for the unit tests I added a separate function set which makes sure that the alternative is shown correctly.

Reply:
Sorry, my IDE for some reason wasn't showing anything... had to delete the project completely.

Reply:
Weird, but at least you got it working now ?